resources/js/bots/wallet/bot.js

```diff
@@ -359,13 +359,17 @@ function validateSend(params, context) {
 function handleSend(params, context) {
     var val = web3.toWei(params["amount"].replace(",", "."), "ether");
 
+    var gasPrice = calculateGasPrice(params["bot-db"]["sliderValue"]);
```

Comment:
do we need this for `0` returned value?

Reply:
I suppose it's always `nil` atm, let's revisit this after release